src/algebra/group/order_synonym.lean

```diff
@@ -0,0 +1,116 @@
+/-
```

Comment:
Should this file be called `order_dual`?

Reply:
It has the material about `lex` as well, further down.

Reply:
Good point. Let's keep it like this.

bors d+